src/hooks/shapediver/viewer/useOutputDrawingTools.ts

```diff
@@ -0,0 +1,112 @@
+import { EVENTTYPE, IGeometryData, IOutputApi, addListener, removeListener } from "@shapediver/viewer";
+import { useEffect, useState } from "react";
+import { CustomizationProperties, IDrawingToolsApi, createDrawingTools } from "@shapediver/viewer.features.drawing-tools";
+import { useShapeDiverStoreViewer } from "store/useShapeDiverStoreViewer";
+import { useParameterStateless } from "../parameters/useParameterStateless";
+import { useOutputContent } from "./useOutputContent";
+
+/**
+ * Hook allowing to create the drawing tools.
+ * 
+ * Makes use of {@link useOutputNode}.
+ * 
+ * @param sessionId 
+ * @param viewportId 
+ * @param outputIdOrName 
+ */
+export function useOutputDrawingTools(sessionId: string, viewportId: string, outputIdOrName: string) : {
+	/**
+	 * API of the output
+	 * @see https://viewer.shapediver.com/v3/latest/api/interfaces/IOutputApi.html
+	 */
+	outputApiCustomizationProperties: IOutputApi | undefined,
+	/**
+	 * API of the drawing tools
+	 */
+	drawingToolsApi: IDrawingToolsApi | undefined
+} {
+	// get the viewport API
+	const viewportApi = useShapeDiverStoreViewer(state => { return state.viewports[viewportId]; });
+	
+	// get the parameter API
+	const parameter = useParameterStateless<string>(sessionId, "points");
+
+	// get the output API and output content for the customization properties
+	const { outputApi: outputApiCustomizationProperties, outputContent: outputContentCustomizationProperties } = useOutputContent(sessionId, outputIdOrName);
```

Comment:
@MajorMeerkatThe3rd thinking about plugging this as a widget into AppBuilder, it will be better if the parameter and the `customizationProperties` are properties of this hook, i.e. you would not call useOutputContent and useParameterStateless in here. Let's talk about it.